packages/grid_client/tests/modules/zmount.test.ts

```diff
@@ -1,58 +1,58 @@
+import { validate } from "class-validator";
+
 import { Zmount } from "../../src";
 
-let zmount: Zmount;
+describe("Zmount Class", () => {
+  let zmount: Zmount;
+  const minSize = 50 * 1024 ** 2; // Less than 100 MB
+  const maxSize = 15 * 1024 ** 4; // Greater than 10 TB
+  const validSize = 5 * 1024 ** 3;
```

Comment:
it's better to have a random size (within a certain range) to test with

you can use this method to generate the random number https://github.com/threefoldtech/tfgrid-sdk-ts/blob/f9a5b06796073bcd61ef729240308d122beb3646/packages/grid_client/src/helpers/utils.ts#L18